Utils/FaviconFetcher.swift

```diff
@@ -27,6 +27,85 @@ class FaviconFetcherErrorType: MaybeErrorType {
 public class FaviconFetcher : NSObject, NSXMLParserDelegate {
     public static var userAgent: String = ""
     static let ExpirationTime = NSTimeInterval(60*60*24*7) // Only check for icons once a week
+    private var defaultIconGlobe = UIImage(named: "defaultFavicon")!
+    private lazy var defaultIconA: UIImage = {
```

Comment:
Why create a separate variable for each one of these? Why not just have a single method that takes a letter, gets the `UIImage` from the cache (or creates and adds it otherwise), then returns that image?